src/legacy/ui/public/chrome/directives/header_global_nav/components/header.tsx

```diff
@@ -248,9 +264,8 @@ class HeaderUI extends Component<Props, State> {
           }),
           listItems: recentlyAccessed.map(item => ({
             label: truncateRecentItemLabel(item.label),
-            // TODO: Add what type of app/saved object to title attr
-            title: `${item.label}`,
-            'aria-label': item.label,
+            title: item.titleAndAriaLabel,
```

Comment:
what about to rename to `title`

I think: 
```
{
...
 title: item.title, 
 'aria-label':  item.title.
...
}
```

should be ok

Reply:
Since that's not totally title, because title comes from a different place. Isn't it? On top of that, now this variable has a speaking name, where you don't need to look through the file in order to understand that, for instance, the "type" word is needed and why.